log_outgoing_requests/models.py

```diff
@@ -60,12 +63,24 @@ class OutgoingRequestsLog(models.Model):
         null=True,
         help_text=_("The request headers."),
     )
+    req_body = models.TextField(
+        verbose_name=_("Request body"),
+        blank=True,
+        null=True,
+        help_text=_("The request body."),
+    )
     res_headers = models.TextField(
         verbose_name=_("Response headers"),
         blank=True,
         null=True,
         help_text=_("The response headers."),
     )
+    res_body = models.JSONField(
```

Comment:
what is the reasoning to make the response body a JSONField while the request body is a TextField?